src/web_crawler/twir_crawler.rs

```diff
@@ -272,3 +294,15 @@ impl TwirCrawler {
         Ok(())
     }
 }
+
+fn get_progress_bar(text: &str) -> ProgressBar {
+    let progress_bar = ProgressBar::new(100);
+    let style = ProgressStyle::default_bar()
+        .template("{prefix} [{bar:40.cyan/blue}] {percent}% {eta} {msg}")
+        .unwrap()
```

Comment:
Can this one fail?

Reply:
It can fail on invalid placeholders (values inside {})

Reply:
All good then, no need to wrap it in a Result, it will probably be uglier like that.